### PR TITLE
Add support for google_compute_healthcheck to TGC

### PR DESCRIPTION
**Pull request by kotarsky**
adding support for compute.googleapis.com/HealthCheck

```release-note:none
```

### CODE REVIEW

**Review by rileykarson — CHANGES_REQUESTED** [review]
See feedback in https://github.com/GoogleCloudPlatform/magic-modules/pull/10540#pullrequestreview-2029564250, can we address that before moving forward with other PRs? Thanks!

**Comment by sushant00** [comment]
Let us update the title according to the recommended format like "Add support for google_compute_autoscaler to TGC"

**Comment by kotarsky** [comment]
> Let us update the title according to the recommended format like "Add support for google_compute_autoscaler to TGC"

@rileykarson changes to title made

File: mmv1/templates/tgc/resource_converters.go.erb [hunk 1]
```diff
@@ -41,6 +41,7 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_compute_forwarding_rule":                  {compute.ResourceConverterComputeForwardingRule()},
 		"google_compute_global_address":                   {compute.ResourceConverterComputeGlobalAddress()},
 		"google_compute_global_forwarding_rule":           {compute.ResourceConverterComputeGlobalForwardingRule()},
+		"google_compute_health_check":           {compute.ResourceConverterComputeHealthCheck()},
```
Comment (rileykarson) [inline mmv1/templates/tgc/resource_converters.go.erb:44]: nit: try to align these in future entries! We run `gofmt` on the output files, but the source should be as close as possible as well.

**Pull request merged**